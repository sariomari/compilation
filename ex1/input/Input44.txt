,})nilzBp0rS95PU"o"=while:=y57fz4k9V5946
void(++<841
if	
=+755
 if]whileif-,=if=;array]	-while-]+	
}nilextends(.<+int/*)
}F
8!;*/16
}((.826
/**/string5
(=7292
}ifreturnextendsclassreturnreturn)[."RdNkn"}>;+=>,{(//!2;;!9!]]5g
{:=while7

3
newifR} {[extendsnew.69
int))/*V?3](Fi;B
);i]]]4}?;1Wvi?*/string]d4t+(80
38
;27
{nilstringvoidnewreturn+new=-]extendsextends//)
{extends<intarray..extends/*(];1o}?]*/(;:=>:=>-new92
6
stringarrayifclass+extendsreturn(while
;""[void//}a
5384
)49
[ ,3

class;+	}5
-}[2310
-""0
;+ifarrayif717
int)[,-intz"wp"}:=22
nil+void}if>extendsvoid+:=413
"dL"class"ZEC"[string4
))return/*5
T;?);;;]};x!
)*/stringvoid9677
+newnil:=intvoid{]int{,+[);>/*L]fgB*/{=	returnclass9
"MzA"while5
string+class=]91
string.string0
if5424
}//sreturn7
intextendswhile5165
,9271
return:=>>//a0newnewnew>GFpt31
}6
J5T
tY/*);aV
P];2)*/new=returnvoidvoid906
,new/*;*/extends[void//]]T;;
:={=//extends250
array<;"Gs"nil[voidwhile{.(->-+;array=	540
array3131
])}extends9951
/*
j;*/("z"<array2235
.[nilextendsnil8
uQ5
array):=]63
70
array-=s3+=whilenilc9>},={S95rMY937NjE:=ifwhile"dfeJ"ifvoid==nilclass7528
"DXeuq"nilk}.intextendsint//O;}?(!9?671
void7631
[>new>,./*;/)3
*/int>;/*](*/array.<,6586
,while=-if""950
+""{48
arrayextends<
.];0
	if=] ,{void=(+"xjf"122
if,nil+9
nil</*
]]*/,17
:=newreturn+641
	:=-870
array7
</*!).d?;1*/extends"n">string:=class//?r
returnnilwhilearrayintwhileint"" +[2535
+)"Puh",{7
return/*[c?;)};]Y
!
!?()!b);3).lL}(;D*/.]return,};>if;{}int,extends"W"=;{class.-/*;9
rGO13!?]?!L?*/-=int-//];;
-/*?!A
(;;X}}]];(S?
*/return671
nilifnilextendsv4]}{arrayextends}[+=///*}}!!*/{nil
if
+intwhile[;;:=,void"CVArr"}returnextends
"sU"}138
.80
-<(,return;while./*?!1[DA;7?);(.((
3.?;}6
!?)(]*/239
<int4397
;jIj4R:=void[(void"mkV"/*to]}s;?}V;k}]*///
 nil]({/**/void}+>;	(-intarray:=<.+//?)Jvr3][}:=;616
if.extends.}nilnil:=63
	,stringreturn{{1400
26
	62
<)string+)},void:=int{void}80
int
A/*]G!;*/array{>-}if/*}?!!])yv*/newwhile)	,)returnnilvoid-92
{,]whilenew-6094
,stringvoid])return=array+{int6
[]whilereturn<.//d(]{(newnilstringif<:=/*]*/}50
int.,nil ;[//intnewvoidint9
-:=[+=:=79
class="Rz"int/*!
?w}
?FC.}G!h7
}/;Q!
*/pnLV[]while+=.nil
intreturnnil{class>classreturnclass/**/-8
//)Wx;]Cintnew+-499
extendsif-int(<=/*n

;5]O0]c5*/	,class;string{arrayclass{int[<new :=stringnilzP554
:=[if../*U]!]I;))


!L??3()!??;4!?2]2?!
!?)}}?}(!5Fx}
w)?;2
C
{T!D*/(whilenilextendsextendsN](arraywhile39
:=:=whilenil8078
.>{[=)>newifreturnnew.voidOS}=(return-class918
4555
.7
}[string2548
-voidint](<)>ifnil{;.>)=int=new+""81
SbQclass-m-408
classarraywhilenewvoidnil9652
>extendsnewop{+{{stringreturn//
int:=new""]extendsvoid/*(!(E}]*/ 7
/*o??(v5
!5]!(}GO!7}?f()!B}]
*/,]//]?};].returnnew	8
{<if(V1M.[{void((,-435
extends.//U)
	extends)}.+class.:=voidnil,newfInewnil(extends/*!!(*/[+}returnstringnilnew-xXiqZ5,"d"nil4668
>=class{new	+string466
<,cZ1{/*;?)(*/void--;return<new"PF""qt"return]return=}=+while<int.4
+//0?v]-;