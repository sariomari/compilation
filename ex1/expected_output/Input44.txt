COMMA[1,1]
RBRACE[1,2]
RPAREN[1,3]
ID(nilzBp0rS95PU)[1,4]
STRING("o")[1,17]
EQ[1,20]
WHILE[1,21]
ASSIGN[1,26]
ID(y57fz4k9V5946)[1,28]
TYPE_VOID[2,1]
LPAREN[2,5]
PLUS[2,6]
PLUS[2,7]
LT[2,8]
INT(841)[2,9]
IF[3,1]
EQ[4,1]
PLUS[4,2]
INT(755)[4,3]
IF[5,2]
RBRACK[5,4]
ID(whileif)[5,5]
MINUS[5,12]
COMMA[5,13]
EQ[5,14]
IF[5,15]
EQ[5,17]
SEMICOLON[5,18]
ARRAY[5,19]
RBRACK[5,24]
MINUS[5,26]
WHILE[5,27]
MINUS[5,32]
RBRACK[5,33]
PLUS[5,34]
RBRACE[6,1]
ID(nilextends)[6,2]
LPAREN[6,12]
DOT[6,13]
LT[6,14]
PLUS[6,15]
TYPE_INT[6,16]
INT(16)[9,5]
RBRACE[10,1]
LPAREN[10,2]
LPAREN[10,3]
DOT[10,4]
INT(826)[10,5]
ID(string5)[11,5]
LPAREN[12,1]
EQ[12,2]
INT(7292)[12,3]
RBRACE[13,1]
ID(ifreturnextendsclassreturnreturn)[13,2]
RPAREN[13,34]
LBRACK[13,35]
DOT[13,36]
STRING("RdNkn")[13,37]
RBRACE[13,44]
GT[13,45]
SEMICOLON[13,46]
PLUS[13,47]
EQ[13,48]
GT[13,49]
COMMA[13,50]
LBRACE[13,51]
LPAREN[13,52]
LBRACE[14,1]
ASSIGN[14,2]
ID(while7)[14,4]
INT(3)[16,1]
ID(newifR)[17,1]
RBRACE[17,7]
LBRACE[17,9]
LBRACK[17,10]
ID(extendsnew)[17,11]
DOT[17,21]
INT(69)[17,22]
TYPE_INT[18,1]
RPAREN[18,4]
RPAREN[18,5]
TYPE_STRING[20,13]
RBRACK[20,19]
ID(d4t)[20,20]
PLUS[20,23]
LPAREN[20,24]
INT(80)[20,25]
INT(38)[21,1]
SEMICOLON[22,1]
INT(27)[22,2]
LBRACE[23,1]
ID(nilstringvoidnewreturn)[23,2]
PLUS[23,24]
NEW[23,25]
EQ[23,28]
MINUS[23,29]
RBRACK[23,30]
ID(extendsextends)[23,31]
LBRACE[24,1]
EXTENDS[24,2]
LT[24,9]
ID(intarray)[24,10]
DOT[24,18]
DOT[24,19]
EXTENDS[24,20]
LPAREN[24,39]
SEMICOLON[24,40]
ASSIGN[24,41]
GT[24,43]
ASSIGN[25,1]
GT[25,3]
MINUS[25,4]
ID(new92)[25,5]
INT(6)[26,1]
ID(stringarrayifclass)[27,1]
PLUS[27,19]
ID(extendsreturn)[27,20]
LPAREN[27,33]
WHILE[27,34]
SEMICOLON[28,1]
STRING("")[28,2]
LBRACK[28,4]
TYPE_VOID[28,5]
INT(5384)[29,1]
RPAREN[30,1]
INT(49)[30,2]
LBRACK[31,1]
COMMA[31,3]
INT(3)[31,4]
CLASS[33,1]
SEMICOLON[33,6]
PLUS[33,7]
RBRACE[33,9]
INT(5)[33,10]
MINUS[34,1]
RBRACE[34,2]
LBRACK[34,3]
INT(2310)[34,4]
MINUS[35,1]
STRING("")[35,2]
INT(0)[35,4]
SEMICOLON[36,1]
PLUS[36,2]
ID(ifarrayif717)[36,3]
TYPE_INT[37,1]
RPAREN[37,4]
LBRACK[39,1]
COMMA[39,2]
MINUS[39,3]
ID(intz)[39,4]
STRING("wp")[39,8]
RBRACE[39,12]
ASSIGN[39,13]
INT(22)[39,15]
NIL[40,1]
PLUS[40,4]
TYPE_VOID[40,5]
RBRACE[41,1]
IF[41,2]
GT[41,4]
ID(extendsvoid)[41,5]
PLUS[41,16]
ASSIGN[41,17]
INT(413)[41,19]
STRING("dL")[42,1]
CLASS[42,5]
STRING("ZEC")[42,10]
LBRACK[42,15]
ID(string4)[42,16]
RPAREN[43,1]
RPAREN[43,2]
RETURN[43,3]
ID(stringvoid9677)[47,4]
PLUS[48,1]
ID(newnil)[48,2]
ASSIGN[48,8]
ID(intvoid)[48,10]
LBRACE[48,17]
RBRACK[48,18]
TYPE_INT[48,19]
LBRACE[48,22]
COMMA[48,23]
PLUS[48,24]
LBRACK[48,25]
RPAREN[48,26]
SEMICOLON[48,27]
GT[48,28]
LBRACE[48,38]
EQ[48,39]
ID(returnclass9)[48,41]
STRING("MzA")[49,1]
ID(while5)[49,6]
TYPE_STRING[50,1]
PLUS[50,7]
CLASS[50,8]
EQ[50,13]
RBRACK[50,14]
INT(91)[50,15]
TYPE_STRING[51,1]
DOT[51,7]
ID(string0)[51,8]
ID(if5424)[52,1]
RBRACE[53,1]
ID(return7)[54,1]
ID(intextendswhile5165)[55,1]
COMMA[56,1]
INT(9271)[56,2]
RETURN[57,1]
ASSIGN[57,7]
GT[57,9]
GT[57,10]
ID(newnewnew)[58,1]
GT[58,10]
ID(GFpt31)[58,11]
RBRACE[59,1]
INT(6)[59,2]
ID(J5T)[60,1]
ID(tY)[61,1]
NEW[63,3]
EQ[63,6]
ID(returnvoidvoid906)[63,7]
COMMA[64,1]
NEW[64,2]
EXTENDS[64,10]
LBRACK[64,17]
TYPE_VOID[64,18]
ASSIGN[65,1]
LBRACE[65,3]
EQ[65,4]
ID(extends250)[66,1]
ARRAY[67,1]
LT[67,6]
SEMICOLON[67,7]
STRING("Gs")[68,1]
NIL[68,5]
LBRACK[68,8]
ID(voidwhile)[68,9]
LBRACE[68,18]
DOT[68,19]
LPAREN[68,20]
MINUS[68,21]
GT[68,22]
MINUS[68,23]
PLUS[68,24]
SEMICOLON[68,25]
ARRAY[68,26]
EQ[68,31]
INT(540)[68,33]
ID(array3131)[69,1]
RBRACK[70,1]
RPAREN[70,2]
RBRACE[70,3]
ID(extends9951)[70,4]
LPAREN[72,5]
STRING("z")[72,6]
LT[72,9]
ID(array2235)[72,10]
DOT[73,1]
LBRACK[73,2]
ID(nilextendsnil8)[73,3]
ID(uQ5)[74,1]
ARRAY[75,1]
RPAREN[75,6]
ASSIGN[75,7]
RBRACK[75,9]
INT(63)[75,10]
INT(70)[76,1]
ARRAY[77,1]
MINUS[77,6]
EQ[77,7]
ID(s3)[77,8]
PLUS[77,10]
EQ[77,11]
ID(whilenilc9)[77,12]
GT[77,22]
RBRACE[77,23]
COMMA[77,24]
EQ[77,25]
LBRACE[77,26]
ID(S95rMY937NjE)[77,27]
ASSIGN[77,39]
ID(ifwhile)[77,41]
STRING("dfeJ")[77,48]
ID(ifvoid)[77,54]
EQ[77,60]
EQ[77,61]
ID(nilclass7528)[77,62]
STRING("DXeuq")[78,1]
ID(nilk)[78,8]
RBRACE[78,12]
DOT[78,13]
ID(intextendsint)[78,14]
INT(671)[79,1]
ID(void7631)[80,1]
LBRACK[81,1]
GT[81,2]
NEW[81,3]
GT[81,6]
COMMA[81,7]
DOT[81,8]
TYPE_INT[82,3]
GT[82,6]
SEMICOLON[82,7]
ARRAY[82,14]
DOT[82,19]
LT[82,20]
COMMA[82,21]
INT(6586)[82,22]
COMMA[83,1]
WHILE[83,2]
EQ[83,7]
MINUS[83,8]
IF[83,9]
STRING("")[84,1]
INT(950)[84,3]
PLUS[85,1]
STRING("")[85,2]
LBRACE[85,4]
INT(48)[85,5]
ID(arrayextends)[86,1]
LT[86,13]
DOT[87,1]
RBRACK[87,2]
SEMICOLON[87,3]
INT(0)[87,4]
IF[88,2]
EQ[88,4]
RBRACK[88,5]
COMMA[88,7]
LBRACE[88,8]
TYPE_VOID[88,9]
EQ[88,13]
LPAREN[88,14]
PLUS[88,15]
STRING("xjf")[88,16]
INT(122)[88,21]
IF[89,1]
COMMA[89,3]
NIL[89,4]
PLUS[89,7]
INT(9)[89,8]
NIL[90,1]
LT[90,4]
COMMA[91,5]
INT(17)[91,6]
ASSIGN[92,1]
ID(newreturn)[92,3]
PLUS[92,12]
INT(641)[92,13]
ASSIGN[93,2]
MINUS[93,4]
INT(870)[93,5]
ID(array7)[94,1]
LT[95,1]
EXTENDS[95,13]
STRING("n")[95,20]
GT[95,23]
TYPE_STRING[95,24]
ASSIGN[95,30]
CLASS[95,32]
ID(returnnilwhilearrayintwhileint)[96,1]
STRING("")[96,31]
PLUS[96,34]
LBRACK[96,35]
INT(2535)[96,36]
PLUS[97,1]
RPAREN[97,2]
STRING("Puh")[97,3]
COMMA[97,8]
LBRACE[97,9]
INT(7)[97,10]
RETURN[98,1]
DOT[101,10]
RBRACK[101,11]
RETURN[101,12]
COMMA[101,18]
RBRACE[101,19]
SEMICOLON[101,20]
GT[101,21]
IF[101,22]
SEMICOLON[101,24]
LBRACE[101,25]
RBRACE[101,26]
TYPE_INT[101,27]
COMMA[101,30]
EXTENDS[101,31]
STRING("W")[101,38]
EQ[101,41]
SEMICOLON[101,42]
LBRACE[101,43]
CLASS[101,44]
DOT[101,49]
MINUS[101,50]
MINUS[103,12]
EQ[103,13]
TYPE_INT[103,14]
MINUS[103,17]
MINUS[104,1]
ID(return671)[107,3]
ID(nilifnilextendsv4)[108,1]
RBRACK[108,18]
RBRACE[108,19]
LBRACE[108,20]
ID(arrayextends)[108,21]
RBRACE[108,33]
LBRACK[108,34]
PLUS[108,35]
EQ[108,36]
LBRACE[110,3]
NIL[110,4]
IF[111,1]
PLUS[112,1]
ID(intwhile)[112,2]
LBRACK[112,10]
SEMICOLON[112,11]
SEMICOLON[112,12]
ASSIGN[112,13]
COMMA[112,15]
TYPE_VOID[112,16]
STRING("CVArr")[112,20]
RBRACE[112,27]
ID(returnextends)[112,28]
STRING("sU")[113,1]
RBRACE[113,5]
INT(138)[113,6]
DOT[114,1]
INT(80)[114,2]
MINUS[115,1]
LT[115,2]
LPAREN[115,3]
COMMA[115,4]
RETURN[115,5]
SEMICOLON[115,11]
WHILE[115,12]
DOT[115,17]
INT(239)[120,4]
LT[121,1]
ID(int4397)[121,2]
SEMICOLON[122,1]
ID(jIj4R)[122,2]
ASSIGN[122,7]
TYPE_VOID[122,9]
LBRACK[122,13]
LPAREN[122,14]
TYPE_VOID[122,15]
STRING("mkV")[122,19]
NIL[124,2]
RBRACK[124,5]
LPAREN[124,6]
LBRACE[124,7]
TYPE_VOID[125,3]
RBRACE[125,7]
PLUS[125,8]
GT[125,9]
SEMICOLON[125,10]
LPAREN[125,12]
MINUS[125,13]
ID(intarray)[126,1]
ASSIGN[126,9]
LT[126,11]
DOT[126,12]
PLUS[126,13]
LBRACK[127,1]
RBRACE[127,2]
ASSIGN[127,3]
SEMICOLON[127,5]
INT(616)[127,6]
IF[128,1]
DOT[128,3]
EXTENDS[128,4]
DOT[128,11]
RBRACE[128,12]
ID(nilnil)[128,13]
ASSIGN[128,19]
INT(63)[128,21]
COMMA[129,2]
ID(stringreturn)[129,3]
LBRACE[129,15]
LBRACE[129,16]
INT(1400)[129,17]
INT(26)[130,1]
INT(62)[131,2]
LT[132,1]
RPAREN[132,2]
TYPE_STRING[133,1]
PLUS[133,7]
RPAREN[133,8]
RBRACE[133,9]
COMMA[133,10]
TYPE_VOID[133,11]
ASSIGN[133,15]
TYPE_INT[133,17]
LBRACE[133,20]
TYPE_VOID[133,21]
RBRACE[133,25]
INT(80)[134,1]
TYPE_INT[135,1]
ID(A)[136,1]
ARRAY[136,10]
LBRACE[136,15]
GT[136,16]
MINUS[136,17]
RBRACE[136,18]
IF[136,19]
ID(newwhile)[137,11]
RPAREN[137,19]
COMMA[137,21]
RPAREN[137,22]
ID(returnnilvoid)[137,23]
MINUS[137,36]
INT(92)[137,37]
LBRACE[138,1]
COMMA[138,2]
RBRACK[138,3]
ID(whilenew)[138,4]
MINUS[138,12]
INT(6094)[138,13]
COMMA[139,1]
ID(stringvoid)[139,2]
RBRACK[139,12]
RPAREN[139,13]
RETURN[139,14]
EQ[139,20]
ARRAY[139,21]
PLUS[139,26]
LBRACE[139,27]
ID(int6)[139,28]
LBRACK[140,1]
RBRACK[140,2]
ID(whilereturn)[140,3]
LT[140,14]
DOT[140,15]
ID(newnilstringif)[141,1]
LT[141,15]
ASSIGN[141,16]
RBRACE[141,23]
INT(50)[141,24]
TYPE_INT[142,1]
DOT[142,4]
COMMA[142,5]
NIL[142,6]
SEMICOLON[142,10]
LBRACK[142,11]
ID(intnewvoidint9)[143,1]
MINUS[144,1]
ASSIGN[144,2]
LBRACK[144,4]
PLUS[144,5]
EQ[144,6]
ASSIGN[144,7]
INT(79)[144,9]
CLASS[145,1]
EQ[145,6]
STRING("Rz")[145,7]
TYPE_INT[145,11]
ID(pnLV)[149,3]
LBRACK[149,7]
RBRACK[149,8]
WHILE[149,9]
PLUS[149,14]
EQ[149,15]
DOT[149,16]
NIL[149,17]
ID(intreturnnil)[150,1]
LBRACE[150,13]
CLASS[150,14]
GT[150,19]
ID(classreturnclass)[150,20]
MINUS[150,40]
INT(8)[150,41]
ID(intnew)[153,1]
PLUS[153,7]
MINUS[153,8]
INT(499)[153,9]
ID(extendsif)[154,1]
MINUS[154,10]
TYPE_INT[154,11]
LPAREN[154,14]
LT[154,15]
EQ[154,16]
COMMA[158,10]
CLASS[158,11]
SEMICOLON[158,16]
TYPE_STRING[158,17]
LBRACE[158,23]
ID(arrayclass)[158,24]
LBRACE[158,34]
TYPE_INT[158,35]
LBRACK[158,38]
LT[158,39]
NEW[158,40]
ASSIGN[158,44]
ID(stringnilzP554)[158,46]
ASSIGN[159,1]
LBRACK[159,3]
IF[159,4]
DOT[159,6]
DOT[159,7]
LPAREN[174,4]
ID(whilenilextendsextendsN)[174,5]
RBRACK[174,28]
LPAREN[174,29]
ID(arraywhile39)[174,30]
ASSIGN[175,1]
ASSIGN[175,3]
ID(whilenil8078)[175,5]
DOT[176,1]
GT[176,2]
LBRACE[176,3]
LBRACK[176,4]
EQ[176,5]
RPAREN[176,6]
GT[176,7]
ID(newifreturnnew)[176,8]
DOT[176,22]
ID(voidOS)[176,23]
RBRACE[176,29]
EQ[176,30]
LPAREN[176,31]
RETURN[176,32]
MINUS[176,38]
ID(class918)[176,39]
INT(4555)[178,1]
DOT[179,1]
INT(7)[179,2]
RBRACE[180,1]
LBRACK[180,2]
ID(string2548)[180,3]
MINUS[181,1]
ID(voidint)[181,2]
RBRACK[181,9]
LPAREN[181,10]
LT[181,11]
RPAREN[181,12]
GT[181,13]
ID(ifnil)[181,14]
LBRACE[181,19]
SEMICOLON[181,20]
DOT[181,21]
GT[181,22]
RPAREN[181,23]
EQ[181,24]
TYPE_INT[181,25]
EQ[181,28]
NEW[181,29]
PLUS[181,32]
STRING("")[181,33]
INT(81)[181,35]
ID(SbQclass)[182,1]
MINUS[182,9]
ID(m)[182,10]
MINUS[182,11]
INT(408)[182,12]
ID(classarraywhilenewvoidnil9652)[183,1]
GT[184,1]
ID(extendsnewop)[184,2]
LBRACE[184,14]
PLUS[184,15]
LBRACE[184,16]
LBRACE[184,17]
ID(stringreturn)[184,18]
TYPE_INT[185,1]
ASSIGN[185,4]
NEW[185,6]
STRING("")[185,9]
RBRACK[185,11]
ID(extendsvoid)[185,12]
INT(7)[187,9]
COMMA[190,3]
RBRACK[190,4]
DOT[192,1]
ID(returnnew)[192,2]
INT(8)[192,12]
LBRACE[193,1]
LT[193,2]
IF[193,3]
LPAREN[193,5]
ID(V1M)[193,6]
DOT[193,9]
LBRACK[193,10]
LBRACE[193,11]
TYPE_VOID[193,12]
LPAREN[193,16]
LPAREN[193,17]
COMMA[193,18]
MINUS[193,19]
INT(435)[193,20]
EXTENDS[194,1]
DOT[194,8]
EXTENDS[195,2]
RPAREN[195,9]
RBRACE[195,10]
DOT[195,11]
PLUS[195,12]
CLASS[195,13]
DOT[195,18]
ASSIGN[195,19]
ID(voidnil)[195,21]
COMMA[195,28]
ID(newfInewnil)[195,29]
LPAREN[195,40]
EXTENDS[195,41]
LBRACK[195,55]
PLUS[195,56]
RBRACE[195,57]
ID(returnstringnilnew)[195,58]
MINUS[195,76]
ID(xXiqZ5)[195,77]
COMMA[195,83]
STRING("d")[195,84]
ID(nil4668)[195,87]
GT[196,1]
EQ[196,2]
CLASS[196,3]
LBRACE[196,8]
NEW[196,9]
PLUS[196,13]
ID(string466)[196,14]
LT[197,1]
COMMA[197,2]
ID(cZ1)[197,3]
LBRACE[197,6]
TYPE_VOID[198,3]
MINUS[198,7]
MINUS[198,8]
SEMICOLON[198,9]
RETURN[198,10]
LT[198,16]
NEW[198,17]
STRING("PF")[198,20]
STRING("qt")[198,24]
RETURN[198,28]
RBRACK[198,34]
RETURN[198,35]
EQ[198,41]
RBRACE[198,42]
EQ[198,43]
PLUS[198,44]
WHILE[198,45]
LT[198,50]
TYPE_INT[198,51]
DOT[198,54]
INT(4)[198,55]
PLUS[199,1]
RBRACK[200,1]
MINUS[200,2]
SEMICOLON[200,3]